src/lib/awsManager.js

```diff
@@ -112,86 +121,87 @@ async function setParametersByService(params, env, service) {
   return data
 }
 
-async function getEnvironments() {
+async function getEnvironments () {
   console.log(`Getting environments descending from ${BASE_PATH}`)
-  var config = {
+  const params = {
     Path: BASE_PATH,
     Recursive: true
-  };
+  }
 
   const envs = {}
   let nextToken = null
 
   initializeSSM()
 
   do {
-    let params = await ssm.getParametersByPath(config).promise()
+    const command = new GetParametersByPathCommand(params)
+    const response = await ssm.send(command)
 
-    for (let i = 0; i < params.Parameters.length; i++) {
-      const param = params.Parameters[i]
+    for (let i = 0; i < response.Parameters.length; i++) {
+      const param = response.Parameters[i]
       const name = param.Name.split('/')
       const env = name[2]
-      envs[env] = envs[env]? envs[env]+1 : 1
+      envs[env] = envs[env] ? envs[env] + 1 : 1
     }
 
-    params = await ssm.getParametersByPath(config).promise()
-    nextToken = params.NextToken
-    config.NextToken = nextToken
```

Comment:
I am not sure why here we invoked `getParametersByPath()` once again - we could have simply used the token from the previous call. Pointing it out just in case it gets missed in the review and if you think there's a valid reason for it.

I have removed the duplicate call for now. Tests seem to pass